tests/test_netstats.py

```diff
@@ -209,3 +234,240 @@ def test_raw_mets():
         print(i)
         print(net_met_val)
         assert net_met_val is not np.nan
+
+
+def test_subgraph_number_of_cliques_for_all():
+    """
+    Test cliques computation
+    """
+    base_dir = str(Path(__file__).parent/"examples")
+    in_mat = np.load(f"{base_dir}/miscellaneous/graphs/002_modality-func_rsn-Default_est-cov_nodety"
+                     f"pe-spheres-2mm_smooth-2fwhm_hpass-0.1Hz_thrtype-PROP_thr-0.95.npy")
+    G = nx.from_numpy_array(in_mat)
+
+    cliques = netstats.subgraph_number_of_cliques_for_all(G)
+
+    assert cliques is not None
+
+
+def test_smallworldness():
+    """
+    Test small-world coefficient (omega) computation
+    """
+    base_dir = str(Path(__file__).parent/"examples")
+    est_path = f"{base_dir}/miscellaneous/0021001_rsn-Default_nodetype-parc_est-sps_thrtype-DENS_thr-0.19.npy"
+
+    in_mat = np.load(est_path)
+    G = nx.from_numpy_array(in_mat)
+
+    omega = netstats.smallworldness(G, niter=5, nrand=5)
+
+    # The dosctring states omega should be between 1 and -1, but it is around -2 here.
```

Comment:
Good catch. If omega is indeed <-1 or >1 then the function must have a bug. I wrote my own version of this manually to handle cases where there are >1 connected subgraphs, which we should parameterize here with two graphs -- 1 with a single connected component, and the other with multiple. We can get this with `if list(nx.connected_components(G)) > 1:`... I can take a deeper look at this one on Monday.

Reply:
Figured out that this was actually correct and the docstring was wrong, because I rewrote the default nx function to handle weighted networks, which should produce a smallworld index > 1